test/browser/spec/ol/source/TileWMS.test.js

```diff
@@ -277,13 +277,11 @@ describe('ol/source/TileWMS', function () {
       expect(uri.hostname).to.be('example.com');
       expect(uri.pathname).to.be('/wms');
       const queryData = uri.searchParams;
-      expect(queryData.get('BBOX')).to.be(
-        '-79.17133464081945,-90,-66.51326044311186,-45'
-      );
+      expect(queryData.get('BBOX')).to.be('-78.75,-67.5,-67.5,-56.25');
       expect(queryData.get('CRS')).to.be('EPSG:4326');
       expect(queryData.get('FORMAT')).to.be('image/png');
       expect(queryData.get('HEIGHT')).to.be('256');
-      expect(queryData.get('I')).to.be('517');
+      expect(queryData.get('I')).to.be('105');
```

Comment:
This test was obiously wrong - with an image `WIDTH` of 256, `I` can never be greater than 256. If fixed the underlying issue, and this is also the reason for the different `BBOX` above, which was incorrect too.